docs/modules/api/pages/index.adoc

```diff
@@ -572,7 +572,7 @@ The link:https://pkg.go.dev/github.com/cerbos/cerbos/client[Cerbos Go SDK] uses
 go get github.com/cerbos/cerbos/api
 ----
 
-You can also make use of the link:https://docs.buf.build/tour/use-remote-generation[remote generation feature of Buf schema registry] to pull down the Cerbos gRPC API as a Go module:
+You can also make use of the link:https://docs.buf.build/generate/remote-plugins[remote plugins feature of Buf schema registry] or the link:https://buf.build/cerbos[generated SDKs] to pull down the Cerbos gRPC API as a Go module:
```

Comment:
```suggestion
You can also make use link:https://buf.build/cerbos/cerbos-api[Buf generated SDKs] to pull down the Cerbos gRPC API as a Go module:
```

Reply:
The `go get` snippet below needs to be updated to `go get buf.build/gen/go/cerbos/cerbos-api/grpc/go@latest` as well.